.github/workflows/ci-weekly.yml

```diff
@@ -47,6 +45,11 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: gcc
+      run: |
```

Comment:
You can remove the `|` if you only have one command to run.  But, when using multiple commands it's required like you have it now.

Reply:
Otherwise, everything looks good so when you're out of draft and test I'll approve

Reply:
I'm changing this up to remove it from the overall tests that are hit with `lerna run test`. Due to the nature of how the compiled test file must be run it's not possible to do it at this time, so I'm removing the requirement until I can add a win32 compatible dlfcn library.